tests/ui/resolve/suggest-unresolve-nightly-149402.rs

```diff
@@ -0,0 +1,9 @@
+//@ only-nightly
```

Comment:
Is it possible to use revisions here to avoid duplicating the same file?

Reply:
thanks, fixed in https://github.com/rust-lang/rust/compare/5b003456246ad18d3141eb748d750f152f51194f..34c80e46389304d906fcb50efe4cf32a0270fab7